app/modules.go

```diff
@@ -0,0 +1,287 @@
+	"github.com/cosmos/cosmos-sdk/x/params"
+	paramsclient "github.com/cosmos/cosmos-sdk/x/params/client"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/cosmos/cosmos-sdk/x/slashing"
+	slashingtypes "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	"github.com/cosmos/cosmos-sdk/x/staking"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/cosmos/cosmos-sdk/x/upgrade"
+	upgradeclient "github.com/cosmos/cosmos-sdk/x/upgrade/client"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	ica "github.com/cosmos/ibc-go/v4/modules/apps/27-interchain-accounts"
+	icatypes "github.com/cosmos/ibc-go/v4/modules/apps/27-interchain-accounts/types"
+	"github.com/cosmos/ibc-go/v4/modules/apps/transfer"
+	ibctransfertypes "github.com/cosmos/ibc-go/v4/modules/apps/transfer/types"
+	ibc "github.com/cosmos/ibc-go/v4/modules/core"
+	ibcclient "github.com/cosmos/ibc-go/v4/modules/core/02-client/client"
+	ibchost "github.com/cosmos/ibc-go/v4/modules/core/24-host"
+	"github.com/persistenceOne/persistence-sdk/v2/x/epochs"
+	epochstypes "github.com/persistenceOne/persistence-sdk/v2/x/epochs/types"
+	"github.com/persistenceOne/persistence-sdk/v2/x/halving"
+	"github.com/persistenceOne/persistence-sdk/v2/x/ibchooker"
+	ibchookertypes "github.com/persistenceOne/persistence-sdk/v2/x/ibchooker/types"
+	"github.com/persistenceOne/persistence-sdk/v2/x/interchainquery"
+	interchainquerytypes "github.com/persistenceOne/persistence-sdk/v2/x/interchainquery/types"
+	"github.com/persistenceOne/pstake-native/v2/x/lscosmos"
+	lscosmosclient "github.com/persistenceOne/pstake-native/v2/x/lscosmos/client"
+	lscosmostypes "github.com/persistenceOne/pstake-native/v2/x/lscosmos/types"
+
+	appparams "github.com/persistenceOne/persistenceCore/v7/app/params"
+)
+
+var ModuleAccountPermissions = map[string][]string{
+	authtypes.FeeCollectorName:               nil,
+	distributiontypes.ModuleName:             nil,
+	icatypes.ModuleName:                      nil,
+	minttypes.ModuleName:                     {authtypes.Minter},
+	stakingtypes.BondedPoolName:              {authtypes.Burner, authtypes.Staking},
+	stakingtypes.NotBondedPoolName:           {authtypes.Burner, authtypes.Staking},
+	govtypes.ModuleName:                      {authtypes.Burner},
+	ibctransfertypes.ModuleName:              {authtypes.Minter, authtypes.Burner},
+	wasm.ModuleName:                          {authtypes.Burner},
+	lscosmostypes.ModuleName:                 {authtypes.Minter, authtypes.Burner},
+	lscosmostypes.DepositModuleAccount:       nil,
+	lscosmostypes.DelegationModuleAccount:    nil,
+	lscosmostypes.RewardModuleAccount:        nil,
+	lscosmostypes.UndelegationModuleAccount:  nil,
+	lscosmostypes.RewardBoosterModuleAccount: nil,
+}
+
+// ModuleBasics defines the module BasicManager is in charge of setting up basic,
+// non-dependant module elements, such as codec registration
+// and genesis verification.
+var ModuleBasics = module.NewBasicManager(
```

Comment:
In osmosis all basics are summarized in `keepers/modules.go`, do you think it seems more appropriate to list them there or also under keepers dir?

Reply:
yup can move it there